runtime/lua/vim/filetype.lua

```diff
@@ -780,19 +780,19 @@ local extension = {
   zsh = "zsh",
   vala = "vala",
   E = function() vim.fn["dist#ft#FTe"]() end,
-  EU = function() vim.fn["dist#ft#EuphoriaCheck"]() end,
-  EW = function() vim.fn["dist#ft#EuphoriaCheck"]() end,
-  EX = function() vim.fn["dist#ft#EuphoriaCheck"]() end,
-  EXU = function() vim.fn["dist#ft#EuphoriaCheck"]() end,
-  EXW = function() vim.fn["dist#ft#EuphoriaCheck"]() end,
+  EU = function(_, bufnr) return require("vim.filetype.detect").euphoria(_, bufnr) end,
```

Comment:
```suggestion
  EU = function(path, bufnr) return require("vim.filetype.detect").euphoria(path, bufnr) end,
```

We want to keep this pattern consistent for now; can be refactored later when the porting is complete.

One thing I don't like is to _pass_  the underscore. I'd prefer if the `detect` function did not take unused arguments at all; the pattern would then be 

`EU = function(path, bufnr) return detect.euphoria(bufnr) end`

Reply:
That seems like a good tradeoff, will do :+1: